### DISCUSSION TITLE
DVCC BMS control of charging devices

### DISCUSSION THREAD

**Discussion by HarrisMarine** [Louisvdw/dbus-serialbattery#651] [Q&A]
Quick question, if I have a bank of say 4 batteries each with its own BMS and only one cell goes into over voltage will that shutdown all the charging devices or will that battery just disconnect and let the others continue to charge?

Also is there a way to control non Victron devices such as a Wakespeed WS500 with DVCC from the Cerbo?

Thanks in advance.

Best regards 
Keith

**Comment by mr-manuel** [comment]
> Quick question, if I have a bank of say 4 batteries each with its own BMS and only one cell goes into over voltage will that shutdown all the charging devices or will that battery just disconnect and let the others continue to charge?

Are you using a battery aggregator? If not, then the controlling BMS selected under DVCC (Venus OS > 3.00) is controlling it for all batteries. Please see the FAQ for a battery aggregator.

> Also is there a way to control non Victron devices such as a Wakespeed WS500 with DVCC from the Cerbo?

1. What do you want to control?
2. What is the device doing? 
3. Which possibilities do you have to control the device?

**Comment by HarrisMarine** [comment]
Hi Mr Manual,

Thanks for your reply, I’m designing a boat install for a client. Usually I just use Victron batteries which have an Allow to Charge and Allow to Discharge terminals on the BMS which can be used for the purpose of controlling non Victron equipment.  But said client has blown part of his battery budget elsewhere (boat does stand for Bring On Another Thousand after all) so he has set me the challenge of integrating a lower cost battery into his system. 
The non Victron items I would like to control are; a Wakespeed WS500 regulated Alternator, a Fischer panda diesel DC generator and a sterling 12v to 24v battery to battery charger.

**Reply by mr-manuel** [reply]
From what I know you can controle the two relais of the Cerbo with Node-RED, but I haven't tried it yet, since I have no Cerbo.
